pkgs/tools/wayland/wayland-proxy-virtwl/default.nix

```diff
@@ -1,39 +1,43 @@
 { lib
 , fetchFromGitHub
 , ocamlPackages
+, pkg-config
+, buildPackages
+, libdrm
 }:
 
 ocamlPackages.buildDunePackage rec {
   pname = "wayland-proxy-virtwl";
-  version = "unstable-2021-12-05";
+  version = "unstable-2022-09-22";
 
   src = fetchFromGitHub {
     owner = "talex5";
     repo = pname;
-    rev = "d7f58d405514dd031f2f12e402c8c6a58e62a885";
-    sha256 = "0riwaqdlrx2gzkrb02v4zdl4ivpmz9g5w87lj3bhqs0l3s6c249s";
+    rev = "5940346db2a4427f21c7b30a2593b179af36a935";
+    sha256 = "0jnr5q52nb3yqr7ykvvb902xsad24cdi9imkslcsa5cnzb4095rw";
   };
 
   postPatch = ''
     # no need to vendor
     rm -r ocaml-wayland
   '';
 
-  useDune2 = true;
-  minimumOCamlVersion = "4.08";
+  minimalOCamlVersion = "4.12";
 
   strictDeps = true;
   nativeBuildInputs = [
-    ocamlPackages.ppx_cstruct
+    buildPackages.ocamlPackages.ppx_cstruct
```

Comment:
Does this not get spliced? D:

Reply:
Oh, it actually does; misremembered that package sets passed as inputs don't get this treatment. Great, I'll just revert this.